Makefile

```diff
@@ -198,6 +201,33 @@ env: ## Print environment variables
 	@echo "IMG=$(IMG)"
 	@echo "CONTAINER_TOOL=$(CONTAINER_TOOL)"
 
+.PHONY: check-typos fix-typos
+check-typos: $(TYPOS) ## Check for spelling errors using typos (exits with error if found)
+	@echo "🔍 Checking for spelling errors with typos..."
+	@TYPOS_OUTPUT=$$($(TYPOS) --format brief 2>&1); \
+	if [ $$? -eq 0 ]; then \
+		echo "✅ No spelling errors found!"; \
+		echo "🎉 Spelling check completed successfully!"; \
+	else \
+		echo "❌ Spelling errors found! \
+		echo "🔧 You can try 'make fix-typos' to automatically fix the spelling errors and run 'make check-typos' again"; \
+		echo "$$TYPOS_OUTPUT"; \
+		exit 1; \
+	fi
+
+fix-typos: $(TYPOS) ## Automatically fix spelling errors using typos
```

Comment:
Q: how confident are we in fixing errors automatically?
Maybe not recommend it and just list the errors?